sky/backends/wheel_utils.py

```diff
@@ -161,13 +163,22 @@ def _get_latest_modification_time(path: pathlib.Path) -> float:
         last_modification_time = _get_latest_modification_time(SKY_PACKAGE_PATH)
         last_wheel_modification_time = _get_latest_modification_time(WHEEL_DIR)
 
-        # only build wheels if the wheel is outdated
-        if last_wheel_modification_time < last_modification_time:
+        # Only build wheels if the wheel is outdated or wheel does not exist
+        # for the requested version.
+        if (last_wheel_modification_time < last_modification_time) or not any(
+                WHEEL_DIR.glob(f'**/{_WHEEL_PATTERN}')):
```

Comment:
Nit: do we need `any` here?

Reply:
Yes, `glob()` returns a generator that we must check with any(). Checking just `glob(f'**/{_WHEEL_PATTERN}')` would always return true since a generator is always returned.